packages/workspace/src/utilities/fileutils.ts

```diff
@@ -47,7 +47,11 @@ export function readJsonFile<T = any>(path: string): T {
 }
 
 export function parseJsonWithComments<T = any>(content: string): T {
-  return JSON.parse(stripJsonComments(content));
+  try {
```

Comment:
Currently it's roughly 5-10% (~35ms to 32ms on nx-incremental-repo single project). It used to be more significant before other optimisations, when we were doing more I/O + parsing.

Based on the benchmark in the official [strip-json-comments](https://github.com/sindresorhus/strip-json-comments) for the average size JSON file (1000 lines) it takes 1-2ms (~800op/s)

Reply:
It's not a significant increase, but most of JSON files don't have comments, so it makes sense to do it as a corner case rather than a happy flow IMHO.

Reply:
Here are the direct comparisons where reading + parsing `nx.json` and `nxdeps.json`:

With `strip-json-comments` as a default operation:
```
0.544791        Read nx.json
68.505417      Read nxdeps.json
```

With `strip-json-comments` as a fallback operation:
```
0.188125        Read nx.json
5.084000       Read nxdeps.json
```

To the lint rule itself in runtime it has no longer a big impact, as most of this heavy lifting is done once on the rule creation.